### PR TITLE
Defining a larger VNET and AKS subnets while allowing for IP space to expand

### PR DESCRIPTION
**Pull request by hatboyzero**
# Defining a larger VNET and AKS subnets while allowing for IP space to expand

## Details on the issue fix or feature implementation

Defining a larger VNET and AKS subnets while allowing for IP space to expand

## Confirm the following

- [ ]  I started this PR by branching from the head of the default branch
- [ ]  I have targeted the PR to merge into the default branch
- [ ]  I have included unit tests for the issue/feature
- [ ]  I have included inline docs for my changes, where applicable
- [ ]  I have successfully run a local build
- [ ]  I have provided the required update scripts, where applicable
- [ ]  I have updated relevant docs, where applicable

### CODE REVIEW

File: deploy/standard/infra/networking-rg.bicep [hunk 1]
```diff
@@ -36,20 +36,22 @@ var privateDnsZone = {
...
 }
 
-var cidrFllmAuth = cidrSubnet(cidrVnet, 26, 17) // 10.220.132.64/26
-var cidrFllmBackend = cidrSubnet(cidrVnet, 24, 1) // 10.220.129.0/24
-var cidrFllmFrontend = cidrSubnet(cidrVnet, 24, 2) // 10.220.130.0/24
-var cidrFllmOpenAi = cidrSubnet(cidrVnet, 26, 12) // 10.220.131.0/26
-var cidrFllmOps = cidrSubnet(cidrVnet, 26, 15) // 10.220.131.192/26
-var cidrFllmVec = cidrSubnet(cidrVnet, 26, 16) // 10.220.132.0/26
-var cidrNetSvc = cidrSubnet(cidrVnet, 24, 6) // 10.220.134.0/24
+var dnsResolverSubnetCidr = cidrSubnet(cidrVnet, 28, 0) // 10.220.134.0/28
```
Comment (jamesrcounts) [inline deploy/standard/infra/networking-rg.bicep:39]: Do you know if we need this here?  Shouldn't it be part of the standard hub?

**Review by jamesrcounts — CHANGES_REQUESTED** [review]

File: deploy/standard/infra/networking-rg.bicep (hunk 1 above)
Comment (hatboyzero) [reply deploy/standard/infra/networking-rg.bicep]: Removed.

File: deploy/standard/infra/modules/vnet-peering.bicep [hunk 2]
```diff
@@ -10,7 +10,7 @@ resource main 'Microsoft.Network/virtualNetworks@2024-01-01' existing = {
 }
 
 resource destinationToSourcePeering 'Microsoft.Network/virtualNetworks/virtualNetworkPeerings@2024-01-01' = {
-  name: 'hub-to-vnet'
+  name: vnetName
```
Comment (jamesrcounts) [inline deploy/standard/infra/modules/vnet-peering.bicep:13]: This should probably be something like 
```suggestion
  name: 'hub-to-${vnetName}`

```

**Review by jamesrcounts — APPROVED** [review]

File: deploy/standard/infra/modules/vnet-peering.bicep (hunk 2 above)
Comment (hatboyzero) [reply deploy/standard/infra/modules/vnet-peering.bicep:13]: Will address in a later PR.

**Review by ciprianjichici — APPROVED** [review]

**Pull request merged**